cookiecutter/wrapper-template/{{cookiecutter.wrapper_id}}/{{cookiecutter.library_name}}/main.py

```diff
@@ -0,0 +1,112 @@
+"""{{ cookiecutter.extension_name }} cli entrypoint."""
+
+import os
+import sys
+from typing import List
+
+import structlog
+import typer
+
+from {{ cookiecutter.library_name }}.extension import {{ cookiecutter.extension_name }}
+from meltano.edk.extension import DescribeFormat
+from meltano.edk.logging import default_logging_config, parse_log_level
+
+APP_NAME: str = "{{ cookiecutter.extension_name }}"
+
+log = structlog.get_logger(APP_NAME)
+
+ext = {{ cookiecutter.extension_name }}()
+
+typer.core.rich = None  # remove to enable stylized help output when `rich` is installed
+app = typer.Typer(pretty_exceptions_enable=False)
+
+
+@app.command()
+def initialize(ctx: typer.Context, force: bool = False) -> None:
+    """Initialize the {{ cookiecutter.extension_name }} plugin.
+
+    Args:
+        ctx: The typer context.
+        force: If True, and if you would like to support it - force initialization.
+    """
```

Comment:
The functions decorated with `app.command()` should not document their args, as their docstrings are provided to the end user as the help text for that command.

```suggestion
def initialize(
    ctx: typer.Context,
    force: bool = typer.Option(False, help="Force initialization (if supported)"),
) -> None:
    """Initialize the {{ cookiecutter.extension_name }} plugin."""
```

Reply:
They're there for extension developers, we definitely should be documenting things for them.  Maybe we just adjust how we comment these.

Reply:
actually , if its just the args - its probably fine to omit them and just maybe document how you might use them later in some not-yet-existing development guide ?